app/models/hvd/Products.scala

```diff
@@ -28,7 +28,7 @@ import cats.data.Validated.{Invalid, Valid}
 
 case class Products(items: Set[ItemType]) {
   def sorted = {
-    items.toSeq.sortBy( it => it.value)
+    items.toSeq.sortBy( it => it.sortIndex)
```

Comment:
```
  private val sortedItemTypes = Seq(Alcohol, Antiques, Caravans, Cars, Clothing, Gold,
    Jewellery, MobilePhones, OtherMotorVehicles, ScrapMetals, Tobacco)
  private def otherValue(items: Set[ItemType]): Option[ItemType] = items.collectFirst { case Other(itemType) => Other(itemType) }
  def sorted = {
    otherValue(items).map { other =>
      sortedItemTypes.intersect(items.toSeq) :+ other
    } getOrElse sortedItemTypes.intersect(items.toSeq)
  }
```

Reply:
@drewmiley, thank you, code refined all tests passing. 
@owen-opencast I have pushed the refined changes if review is required again. 
Thanks